subsys/bluetooth/services/fast_pair/fp_registration_data.c

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
The data is provisioned to device once and it's not supposed to change. It would have to be moved between FLASH pages by settings subsystem when settings pages are erased. There is no need to do it.

Reply:
For security reasons it's even recommended to store the private key using a Secure Element (SE). In our case we use internal FLASH where we could disable ability to read the data after device is released.

Reply:
Ok. It explain this to me